app/controllers/concerns/bbb_server.rb

```diff
@@ -54,6 +54,7 @@ def join_path(room, name, options = {}, uid = nil)
     join_opts = {}
     join_opts[:userID] = uid if uid
     join_opts[:join_via_html5] = true
+    join_opts[:avatarURL] = options[:avatarURL] if options[:avatarURL].present? && options[:avatarURL_size] < ENV['AVATAR_SIZE_LIMIT'].to_i
```

Comment:
Can we move the logic for validating the image size into the controller instead? 

```
join_opts[:avatarURL] = options[:avatarURL] if options[:avatarURL].present?
```